streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -4560,6 +4560,12 @@ object ZStream extends ZStreamPlatformSpecificConstructors {
   def concatAll[R, E, O](streams: => Chunk[ZStream[R, E, O]])(implicit trace: ZTraceElement): ZStream[R, E, O] =
     ZStream.suspend(streams.foldLeft[ZStream[R, E, O]](empty)(_ ++ _))
 
+  /**
+   * Prints the specified message to the console for debugging purposes.
+   */
+  def debug(value: => Any)(implicit trace: ZTraceElement): ZStream[Any, Nothing, Unit] =
+    ZStream.fromZIO(ZIO.debug(println(value)))
```

Comment:
why println? ZIO.debug already takes care of printing?

Reply:
Good catch! Fixed now.